src/components/pages/EdcConnector/AddConnectorOverlay/index.tsx

```diff
@@ -93,40 +111,71 @@ const AddConnectorOverlay = ({
   }, [openDialog, reset])
 
   const onFormSubmit = async () => {
-    const validateFields = await trigger([
-      'ConnectorName',
-      'ConnectorURL',
-      'ConnectorLocation',
-      'ConnectorSubscriptionId',
-    ])
+    const validateFields =
+      newTechnicalUSer && !newUserSuccess
+        ? await trigger(['TechnicalUserName', 'TechnicalUserDescription'])
+        : await trigger([
+            'ConnectorName',
+            'ConnectorURL',
+            'ConnectorLocation',
+            'ConnectorSubscriptionId',
+            'ConnectorTechnicalUser',
+          ])
     if (validateFields) {
-      onFormConfirmClick(getValues() as FormFieldsType)
+      newTechnicalUSer && !newUserSuccess
+        ? onSubmitClick(getValues() as FormFieldsType)
+        : onFormConfirmClick(getValues() as FormFieldsType)
     }
   }
 
   const onSelect = (service: ConnectorType) => {
     setSelected(service)
   }
 
+  const getTitle = () => {
+    if (connectorStep === 1 && selected.type === 'MANAGED_CONNECTOR')
+      return t('content.edcconnector.modal.managed.title')
+    else if (
+      connectorStep === 1 &&
+      selected.type === ConnectType.COMPANY_CONNECTOR
+    ) {
+      return t('content.edcconnector.modal.company.title')
+    } else return t('content.edcconnector.modal.title')
+  }
+  const getIntro = () => {
+    if (
+      connectorStep === 1 &&
+      selected.type === ConnectType.MANAGED_CONNECTOR
+    ) {
+      return t('content.edcconnector.modal.managed.intro')
+    } else if (
+      connectorStep === 1 &&
+      selected.type === ConnectType.COMPANY_CONNECTOR
+    ) {
+      return (
+        <Typography variant="body1">
```

Comment:
any reason why only this is wrapped in a separate <Typograpy> tag?

Reply:
Yes, this is a big description and was asked to change for better look and feel.